packages/core/src/lib/markdown-it/index.js

```diff
@@ -7,9 +7,16 @@ const slugify = require('@sindresorhus/slugify');
 
 const { HighlightRule } = require('./highlight/HighlightRule.js');
 
+const createDoubleDelimiterInlineRule = require('./markdown-it-double-delimiter');
+
 // markdown-it plugins
+
+markdownIt.use(createDoubleDelimiterInlineRule('%%', 'dimmed', 'emphasis'))
+  .use(createDoubleDelimiterInlineRule('$$', 'underline', 'dimmed'))
+  .use(createDoubleDelimiterInlineRule('++', 'large', 'underline'))
+  .use(createDoubleDelimiterInlineRule('--', 'small', 'large'));
+
 markdownIt.use(require('markdown-it-mark'))
-  .use(require('markdown-it-ins'))
```

Comment:
I'm not really well versed on `markdown-it-ins` and its supposed usage in our project, was it related to the double-delimiter plugin? If not, I think it's better to address this in a code maintenance PR, to keep the changes here about the new syntax.

If we end up really not use the plugin anymore, we can also remove it from our dependencies in `package.json` as well.

Reply:
`markdown-it-ins` is actually to enable the `++` syntax for underlining. 

Since in this PR we are using `++` for larger text and `$$` for underlining instead, this plugin is no longer needed. And I think it's better to remove it since it may interfere with the parsing for the `++` syntax. 

Thus, I'm not sure if we should address it here or in a code maintenance PR :-/ 

Yup, I think it should be removed from `package.json` as well, but would need to seek advice from @ang-zeyu for it.

Reply:
Oh I see, I thought it was used for other purposes. I feel it can be kept here. :+1:

Reply:
let's remove it (plugin), since we aren't using it anywhere else